lib/rspec/page-regression/matcher.rb

```diff
@@ -9,10 +9,15 @@ module RSpec::PageRegression
 
       @responsive_filepaths = FilePaths.responsive_file_paths(RSpec.current_example, args)
 
-      opt = args.select { |k,_| RENDER_ARGS.include?(k) }
-
-      Renderer.render_responsive(page, @responsive_filepaths, opt)
-      @comparisons = @responsive_filepaths.map{ |filepaths| ImageComparison.new(filepaths) }
+      render_options = args.select { |k,_| RENDER_ARGS.include?(k) }
+      compare_options = {}
+      compare_options[:threshold] = args.fetch(:threshold, RSpec::PageRegression.threshold)
+      compare_options[:mode] = args.fetch(:mode, RSpec::PageRegression.mode)
+      compare_options[:include_rect] = args.fetch(:include_area, nil)
+      compare_options[:exclude_rect] = args.fetch(:exclude_rect, nil)
```

Comment:
@ronen I see your point. However for consistency I'd prefer to either have all imatcher args wrapped in an imatcher hash or have all imatcher args flattened into the main args hash. As we have a well defined dependency to imatcher (`gem 'imatcher', '~> 0.1.9'`) it would be fine for me to hardcode all known imatcher options of the according imatcher version. This also protects us from users who try to use imatcher options which are incompatible with our gem (nobody can foresee which options might be supported by imatcher in future).

Also we should replace `include_area` by `include_rect` in line 16.